src/golang/cmd/server/handler/preview.go

```diff
@@ -50,28 +49,9 @@ type previewArgs struct {
 	// Add list of IDs
 }
 
-type previewFloatArtifactResponse struct {
-	Val float64 `json:"val"`
-}
-
-type previewBoolArtifactResponse struct {
-	Passed bool `json:"passed"`
-}
-
-type previewParamArtifactResponse struct {
-	Val string `json:"val"`
-}
-
-type previewTableArtifactResponse struct {
-	TableSchema []map[string]string `json:"table_schema"`
```

Comment:
just to confirm, we're intentionally getting rid of TableSchema here right?

Reply:
This info doesn't seem to be used by the SDK, so I got rid of this.